README.md

```diff
@@ -1,2 +1,10 @@
 # speakbox-fall-2018
+
 Wiki: [Simplified GIT Workflow](https://github.com/redacademy/speakbox-fall-2018/wiki/Simplified-GIT-Workflow)
```

Comment:
This wiki info is not part of your README. Please remove this wiki info, it is for you know information only. If you want to get this info, go to wiki tab and then click on the Simplified GIT Workflow page.